showcase/src/assets/translations/fr.json

```diff
@@ -330,6 +330,7 @@
         },
         "REGULAR": "Table régulière",
         "WITH_SELECTION": "Table avec sélection",
+        "WITH_ITEMS_PER_PAGE_SELECTOR": "Table avec la sélection du nombre d'éléments à afficher",
```

Comment:
I suggest to change this for:
"Table avec le sélecteur du nombre d'éléments par page à afficher"